pytest_selenium/driver.py

```diff
@@ -44,9 +47,13 @@ def make_driver(item):
 
 def start_driver(item, capabilities):
     options = item.config.option
-    specific_driver = '%s_driver' % options.driver.lower()
-    driver_method = globals().get(specific_driver, generic_driver)
-    driver = driver_method(item, capabilities)
+    driver_name = (options.driver and options.driver.lower()
+                   or None)
+    if driver_name not in [browser.lower() for browser in SUPPORTED_DRIVERS]:
```

Comment:
The problem with not creating a variable that stores the value of `options.driver and options.driver.lower()` is that on a later line we need to use `options.driver` again, so we'd need to do that check for `None` again. I'd rather do it once, which is why I created the variable.